### ISSUE TITLE
`python3 ./mach build` causes errors when using pip's new resolver

### ISSUE THREAD

**Issue by duckinator** [servo/servo#27727]
## Problem

For anyone using [pip's new resolver](https://pip.pypa.io/en/stable/user_guide/#changes-to-the-pip-dependency-resolver-in-20-2-2020) **which will become default in pip 20.3**, running `python3 ./mach build --release` will result in this error:

```
The conflict is caused by:
    The user requested voluptuous==0.10.5
    servo-tidy 0.3.0 depends on voluptuous==0.11.5

To fix this you could try to:
1. loosen the range of package versions you've specified
2. remove package versions to allow pip attempt to solve the dependency conflict

Error:
ERROR: Cannot install voluptuous==0.10.5 and -r /home/puppy/dev/rust/servo/servo/python/requirements.txt (line 39) because these package versions have conflicting dependencies.
ERROR: ResolutionImpossible: for help visit https://pip.pypa.io/en/latest/user_guide/#fixing-conflicting-dependencies
```

## Cause

The problem is caused by requirements in both `python/requirements.txt` and `python/tidy/setup.py` that specify the same package, but different versions. When installed into the same environment, pip's old resolver quietly ignores this problem, but the new resolver (correctly) throws an error.

## Solution

The short-term solution is to just make the versions in the two files match.

Ideally there would only be one place where the versions are specified, but I'm not sure how to approach that.

**Comment by ghostd** [comment]
It seems related to #10611

**Comment by jdm** [comment]
Yeah, #10611 is long-term solution that @duckinator mentioned.

**Comment by jdm** [comment]
This should be fixed by #27881.
